packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -41,15 +40,15 @@ export const styles = (theme) => ({
     display: 'flex',
     width: '100%',
     '&$horizontal': {
-      width: 'initial',
```

Comment:
https://stackoverflow.com/questions/20823105/use-initial-width-for-element-not-working-in-ie